src/ducks/apps/components/MyApplications.jsx

```diff
@@ -14,49 +13,55 @@ export class MyApplications extends Component {
   }
 
   onAppClick (appSlug) {
-    this.props.history.push(`/myapps/${appSlug}/manage`)
+    this.props.history.push(`/myapps/${appSlug}`)
   }
 
   render () {
     const { t, installedApps, isFetching, fetchError } = this.props
     return (
       <div className='sto-myapps'>
-        <h2 className='sto-myapps-title'>{t('myapps.title')}</h2>
-        <div className='sto-myapps-list'>
-          {!isFetching && installedApps && !!installedApps.length &&
-            installedApps.map(app => {
-              return <SmallAppItem
-                slug={app.slug}
-                developer={app.developer}
-                editor={app.editor}
-                icon={app.icon}
-                name={app.name}
-                version={app.version}
-                installed={app.installed}
-                onClick={() => this.onAppClick(app.slug)}
-                key={app.slug}
-              />
-            })
-          }
-          {fetchError &&
-            <p className='coz-error'>{fetchError.message}</p>
-          }
-        </div>
+        {
+          this.props.match.isExact
+          ? <div>
+            <h2 className='sto-myapps-title'>{t('myapps.title')}</h2>
+            <div className='sto-myapps-list'>
+              {!isFetching && installedApps && !!installedApps.length &&
+                installedApps.map(app => {
+                  return <SmallAppItem
+                    slug={app.slug}
+                    developer={app.developer}
+                    editor={app.editor}
+                    icon={app.icon}
+                    name={app.name}
+                    version={app.version}
+                    installed={app.installed}
+                    onClick={() => this.onAppClick(app.slug)}
+                    key={app.slug}
+                  />
+                })
+              }
+              {fetchError &&
+                <p className='coz-error'>{fetchError.message}</p>
+              }
+            </div>
+          </div>
+          : null
+        }
+
+        <ApplicationRouting
+          t={t}
```

Comment:
You don't need to provide the `t` here since it's provided by the `translate()` HOC [here](https://github.com/cozy/cozy-store/pull/18/files#diff-4b1cc606dd3a6ffb95d77507b8db66faR37)